Source/Shared/Storage/HybridStorage.swift

```diff
@@ -61,6 +61,15 @@ extension HybridStorage: StorageAware {
     notifyStorageObservers(about: .add(key: key))
   }
 
+  public func existsObject(forKey key: String) throws -> Bool {
```

Comment:
With this implementation `throws` is not required anymore as `DiskStorage` and `MemoryStorage` doesn't throw errors explicitly anymore.

Instead we could write following but it would break API backward compatibility:
```suggestion
      public func existsObject(forKey key: String) -> Bool {
         return try memoryStorage.existsObject(forKey: key) ||
                         diskStorage.existsObject(forKey: key)
      }
```